content/en/events/kcseu/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Contributor Summit Europe 2022"
+type: docs
+aliases: [ "/summit", "/kcseu2022" ]
+weight: 1
+description: >
+  Kubernetes Contributor Summit Europe, hosted in Valencia Spain.
+
+---
+
+
+### About
+
+The Europe Summit will be an lightweight event taking place on May 16th
+in Valencia, Spain alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>.
+
+Similar to North America summit last year, the summit in Europe will be
+informal with minimal programming, but plenty of space to hang out, catch up,
+and meet new contributors.
+
+This page will be updated with more information as it develops. If you have
+questions or comments, please reach out to the Summit Staff by emailing
+community@kubernetes.io or asking in the
+<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">#contributor-summit</a>
+slack channel.
+
+[location]: /events/kcseu/location/
+[RSVP]: /events/kcseu/registration/
+
+
+
+### Code of Conduct
+
+Attendees agree to abide by Kubernetes [Code of Conduct]. More information on
+how to report an event will be posted in the future.
```

Comment:
Would it make sense to replace it with "a violation or suspected violation"?

Reply:
We should already know this.  Let's link to the main kubecon site.